docs/user_docs/installation/install-with-kbcli/install-kbcli.md

```diff
@@ -42,12 +42,18 @@ You can install kbcli with `curl` or `brew`.
       ```bash
       curl -fsSL https://kubeblocks.io/installer/install_cli.sh | bash -s x.x.x
       ```
+   You can view all versions of kbcli, including alpha and beta releases, on the [kbcli releases list](https://github.com/apecloud/kbcli/releases).
+
+   To get stable releases, use this command:
+   ```bash
+   curl -s "https://api.github.com/repos/apecloud/kbcli/releases?per_page=100&page=1" | jq '.[] | select(.prerelease == false) | .tag_name'
```

Comment:
Recommend to use this pipe `| jq -r '.[] | select(.prerelease == false) | .tag_name' | sort -V` for better result.

```bash
$ curl -s "https://api.github.com/repos/apecloud/kubeblocks/releases?per_page=100&page=1" | jq -r '.[] | select(.prerelease == false) | .tag_name' | sort -V
v0.5.1
v0.5.2
v0.5.3
v0.6.0
v0.6.1
v0.6.2
v0.6.3
v0.6.4
v0.7.0
v0.7.1
v0.7.2
v0.8.0
v0.8.1
v0.8.2
v0.8.3
v0.8.4
v0.9.0
```

Reply:
great!